site/specs-source/voice.json

```diff
@@ -49,7 +49,7 @@
           "Calls"
         ],
         "summary": "Create Call",
-        "description": "Creates an outbound phone call.",
+        "description": "Creates an outbound phone call.\n\nCalls are created immediately unless your account has outbound call queueing enabled. When enabled, your outbound calls will be queued and initiated at a specific dequeueing rate, enabling your application to \"fire and forget\" when creating calls. Queued calls may not be modified until they are dequeued and placed, but may be removed from your queue on demand.\n\n<b>Please note: </b> Calls submitted to your queue will be placed aproximately in order, but exact ordering is not guaranteed.",
```

Comment:
I think there should be a note about how to get call queueing enabled for your account

Reply:
And probably a link to the limits page so people know how to increase their queue rates

Reply:
Hmmmm maybe add the stuff about enabling call queuing on the limits page?